library/direct_message_check.inc

```diff
@@ -44,7 +44,7 @@ function phimail_connect(&$phimail_error)
 
     $phimail_server = @parse_url($GLOBALS['phimail_server_address']);
     $phimail_username = $GLOBALS['phimail_username'];
-    $phimail_password = $GLOBALS['phimail_password'];
+    $phimail_password = decryptStandard($GLOBALS['phimail_password']);
```

Comment:
btw, the above comment is the only thing you need to address. then this will be ready for the codebase.

Reply:
Okay, will do. I must have somehow missed that.

Reply:
Huh, apparently I didn't miss that. It just wasn't uploaded when I did the commit for some reason. I'm going to try and figure out why that is.

Reply:
When I'm in the terminal, I run git commit -a and get on branch oemr and untracked files as well as nothing added to commit but untracked files present. 

../interface/weno/#jsonScript.php#
#globals.inc.php#

These are the two untracked files so direct_message_check.inc seems to be tracked. When I try git push origin oemr I get Everything up-to-date, but when I actually go to my github and check direct_message_check.inc that doesn't match what I have saved locally. Any ideas on what would be causing that and why it would only seemingly have issues with one file?

Reply:
@JonathanKoester , what do you see for:
```
git  log
```
And what does your .git/config look like?

Reply:
git config --list:

>user.name=JonathanKoester
user.email=jonskoester@gmail.com
core.editor=emacs
core.excludesfile=~/.gitignore
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
remote.origin.url=https://github.com/JonathanKoester/openemr.git
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.master.remote=origin
branch.master.merge=refs/heads/master
remote.upstream.url=git://github.com/openemr/openemr.git
remote.upstream.fetch=+refs/heads/*:refs/remotes/upstream/*

git log:

> commit bb87894ee4eb6e28461c5ff1991b2a1429d99b5f
Author: JonathanKoester <jonskoester@gmail.com>
Date:   Sun Jan 6 21:34:59 2019 -0600

    Added encryption and decryptStandard to globals

commit 54bc980dc64379d1a7cada127562d07dd9f82483
Author: Brady Miller <brady.g.miller@gmail.com>
Date:   Wed Jan 2 11:39:53 2019 -0800

    minor fix to prior encryption work

Reply:
Very odd. We could just chalk it up to initial git weirdness and I could bring this PR into codebase now and I'll do a quick fix in a commit following it :)
As an aside:
Why do you have those untracked files?
What dev environment are you using?

Reply:
I have a VM running linux mint. I mostly just use emacs for everything. I'm not sure where those files come from. They seem to be duplicates of files that actually exist. I'm not sure how or why they are created or why specifically only for some files they are created.

Reply:
what do you see with:
```
git log origin/oemr
```

Reply:
actually, no need to do above. I was able to do it. Your local direct_message_check.inc shoudl definitely be the same as the one on github. This is very odd.

Reply:
Maybe it's somem weirf emacs buffer weirdness. Try to edit and save it in another text editor,

Reply:
When I switch the branch on my github page to oemr, it is showing up now.